datasets/autshumato_parallel/autshumato_parallel.py

```diff
@@ -0,0 +1,208 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Autshumato Parallel Corpora"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{groenewald2010processing,
+  title={Processing parallel text corpora for three South African language pairs in the Autshumato project},
+  author={Groenewald, Hendrik J and du Plooy, Liza},
+  journal={AfLaT 2010},
+  pages={27},
+  year={2010}
+}
+"""
+
+_DESCRIPTION = """\
+Multilingual information access is stipulated in the South African constitution. In practise, this
+is hampered by a lack of resources and capacity to perform the large volumes of translation
+work required to realise multilingual information access. One of the aims of the Autshumato
+project is to develop machine translation systems for three South African languages pairs.
+"""
+
+
+class AutshumatoConfig(datasets.BuilderConfig):
+    """ BuilderConfig for NewDataset"""
+
+    def __init__(self, langs, zip_file, **kwargs):
+        """
+
+        Args:
+            pair: the language pair to consider
+            zip_file: The location of zip file containing original data
+            **kwargs: keyword arguments forwarded to super.
+        """
+        self.langs = langs
+        self.zip_file = zip_file
+        super().__init__(**kwargs)
+
+
+class Autshumato(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class AutshumatoParallel(datasets.GeneratorBasedBuilder):
```
this needs to be the camel case version of the dataset script name `autshumato_parallel.py`

After this change you'll need to regenerate the dataset_infos.json file using the cli